examples/uniaxial_extension/uniaxial_extension.jl

```diff
@@ -7,90 +7,114 @@ using ONSAS
+    n2 = Node(0.0, 0.0, Lz)
+    n3 = Node(0.0, Ly, Lz)
+    n4 = Node(0.0, Ly, 0.0)
+    n5 = Node(Lx, 0.0, 0.0)
+    n6 = Node(Lx, 0.0, Lz)
+    n7 = Node(Lx, Ly, Lz)
+    n8 = Node(Lx, Ly, 0.0)
+    vec_nodes = [n1, n2, n3, n4, n5, n6, n7, n8]
+    m = Mesh(; nodes=vec_nodes)
     ## Faces
-    f₁ = TriangularFace(n₅, n₈, n₆, "loaded_face_1")
-    f₂ = TriangularFace(n₆, n₈, n₇, "loaded_face_2")
-    f₃ = TriangularFace(n₄, n₁, n₂, "x=0_face_1")
-    f₄ = TriangularFace(n₄, n₂, n₃, "x=0_face_2")
-    f₅ = TriangularFace(n₆, n₂, n₁, "y=0_face_1")
-    f₆ = TriangularFace(n₆, n₁, n₅, "y=0_face_2")
-    f₇ = TriangularFace(n₁, n₄, n₅, "z=0_face_1")
-    f₈ = TriangularFace(n₄, n₈, n₅, "z=0_face_2")
-    vec_faces = [f₁, f₂, f₃, f₄, f₅, f₆, f₇, f₈]
-    append!(faces(s₁_mesh), vec_faces)
+    f1 = TriangularFace(n5, n8, n6, "loaded_face_1")
+    f2 = TriangularFace(n6, n8, n7, "loaded_face_2")
+    f3 = TriangularFace(n4, n1, n2, "x=0_face_1")
+    f4 = TriangularFace(n4, n2, n3, "x=0_face_2")
+    f5 = TriangularFace(n6, n2, n1, "y=0_face_1")
+    f6 = TriangularFace(n6, n1, n5, "y=0_face_2")
+    f7 = TriangularFace(n1, n4, n5, "z=0_face_1")
+    f8 = TriangularFace(n4, n8, n5, "z=0_face_2")
+    vec_faces = [f1, f2, f3, f4, f5, f6, f7, f8]
+    append!(faces(m), vec_faces)
     ## Entities
-    t₁ = Tetrahedron(n₁, n₄, n₂, n₆, "tetra_1")
-    t₂ = Tetrahedron(n₆, n₂, n₃, n₄, "tetra_2")
-    t₃ = Tetrahedron(n₄, n₃, n₆, n₇, "tetra_3")
-    t₄ = Tetrahedron(n₄, n₁, n₅, n₆, "tetra_4")
-    t₅ = Tetrahedron(n₄, n₆, n₅, n₈, "tetra_5")
-    t₆ = Tetrahedron(n₄, n₇, n₆, n₈, "tetra_6")
-    vec_elems = [t₁, t₂, t₃, t₄, t₅, t₆]
-    append!(elements(s₁_mesh), vec_elems)
+    t1 = Tetrahedron(n1, n4, n2, n6, "tetra_1")
+    t2 = Tetrahedron(n6, n2, n3, n4, "tetra_2")
+    t3 = Tetrahedron(n4, n3, n6, n7, "tetra_3")
+    t4 = Tetrahedron(n4, n1, n5, n6, "tetra_4")
+    t5 = Tetrahedron(n4, n6, n5, n8, "tetra_5")
+    t6 = Tetrahedron(n4, n7, n6, n8, "tetra_6")
+    vec_elems = [t1, t2, t3, t4, t5, t6]
+    append!(elements(m), vec_elems)
     # -------------------------------
     # Dofs
     #--------------------------------
     dof_dim = 3
     dof_u_symbol = :u
-    set_dofs!(s₁_mesh, dof_u_symbol, dof_dim)
+    set_dofs!(m, dof_u_symbol, dof_dim)
     # -------------------------------
     # Materials
     # -------------------------------
     svk = SVK(; E=E, ν=ν, label="svk")
-    s₁_materials = StructuralMaterial(svk => [t₁, t₂, t₃, t₄, t₅, t₆])
+    mat = StructuralMaterial(svk => [t1, t2, t3, t4, t5, t6])
     # -------------------------------
     # Boundary conditions
     # -------------------------------
     # Fixed dofs
-    bc₁_label = "fixed-ux"
-    bc₁ = FixedDof(:u, [1], bc₁_label)
-    bc₂_label = "fixed-uj"
-    bc₂ = FixedDof(:u, [2], bc₂_label)
-    bc₃_label = "fixed-uk"
-    bc₃ = FixedDof(:u, [3], bc₃_label)
+    bc1_label = "fixed-ux"
+    bc1 = FixedDof(:u, [1], bc1_label)
+    bc2_label = "fixed-uj"
```

Comment:
```suggestion
    bc2_label = "fixed-uy"
```

Reply:
The final resolution was clarified in [this comment](https://github.com/ONSAS/ONSAS.jl/pull/495#discussion_r1660223768)